README.md

```diff
@@ -1,24 +1,88 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Budget-App
 
-Things you may want to cover:
+## Description
 
-* Ruby version
+In this project, I created a budget website in which users can track their finances and what they're being spent on. 
 
-* System dependencies
+## Screenshots
 
-* Configuration
+![image](https://user-images.githubusercontent.com/74814780/159991267-1dcae1b2-f4a4-4837-be7a-dc4877926c5b.png)
 
-* Database creation
+![image](https://user-images.githubusercontent.com/74814780/159991298-b0889634-e711-4676-89e0-fc7f0d271367.png)
 
```

Comment:
- As required earlier in this PR, please deploy your app and include the live link to your README. This is the last step to get the approval, thank you for your efforts so far ❤️🙏🏻😁 Keep up 👆